homeassistant/components/homeconnect/manifest.json

```diff
@@ -0,0 +1,13 @@
+{
+  "domain": "homeconnect",
+  "name": "Home Connect",
+  "documentation": "https://www.home-assistant.io/integrations/homeconnect",
+  "dependencies": ["http"],
+  "codeowners": [
+    "@DavidMStraub"
+  ],
+  "requirements": [
+    "homeconnect==0.3.2"
```

Comment:
I think you forgot to release version `0.3.2` of `homeconnect` before this 😁

Reply:
Oh dear, I don't know what I was doing there :flushed: it should be 0.3.1, actually. Thanks, corrected!

Reply:
No problem at all!